app/views/steps/shared/_existing_names.html.erb

```diff
@@ -1,8 +1,16 @@
 <%= person.hidden_field :id %>
 
-<% if names_form.object.split_names? %>
-  <%= person.text_field :first_name, class: 'narrow' %>
-  <%= person.text_field :last_name, class: 'narrow' %>
-<% else %>
-  <%= person.text_field :full_name %>
-<% end %>
+<fieldset class="fieldset-compact">
+  <% if legend %>
```

Comment:
I personally would get rid of this IF because in all instances we always pass a `legend` and I don't anticipate not having ever a legend in this partial, as it is only used in the names views.

Reply:
I was thinking more for safety and then the possibility of using this partial for new names too.

I'll get rid of it for now though